fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBJEJournal.java

```diff
@@ -375,10 +375,24 @@ private void reSetupBdbEnvironment(InsufficientLogException insufficientLogEx) {
         // ATTN: here we use `getServingEnv()`, because only serving catalog has
         // helper nodes.
         HostInfo helperNode = Env.getServingEnv().getHelperNode();
-        NetworkRestore restore = new NetworkRestore();
-        NetworkRestoreConfig config = new NetworkRestoreConfig();
-        config.setRetainLogFiles(false);
-        restore.execute(insufficientLogEx, config);
+
+        for (int i = 0; i < RETRY_TIME; i++) {
+            try {
+                NetworkRestore restore = new NetworkRestore();
+                NetworkRestoreConfig config = new NetworkRestoreConfig();
+                config.setRetainLogFiles(false);
+                restore.execute(insufficientLogEx, config);
```

Comment:
I'm not familiar with the logic here, but it looks like you're retrying up to the maximum number of times, even if it was successful before then.

Reply:
> I'm not familiar with the logic here, but it looks like you're retrying up to the maximum number of times, even if it was successful before then.

Thanks for your comment, i have added a `break` here.